src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -2,10 +2,13 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
-  http://www.springframework.org/schema/beans/spring-beans.xsd
-  http://www.springframework.org/schema/context
-  http://www.springframework.org/schema/context/spring-context.xsd"
->
-    <bean id="greetingService" class="org.noteskeeper.server.GreetingService"/>
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
```

Comment:
Spring XML configs are still widely used (and has its own advantages) and it's a good starting point.
anyway, spring beans 2.5 version schema is extremely outdated. mkyoung example uses spring 3 and relies on dep jersey-spring3 which looks strange with spring 4.
there's no similar dep for spring 4 yet => jersey JAR-RS implementation looks outdated / not actively supported.
I'd strongly recommend to switch to Apache CXF instead of Jersey, as it support Spring better